client/components/CardEntry.jsx

```diff
@@ -59,38 +73,44 @@ const Description = styled.div`
   letter-spacing: .013em;
 `;
 
-// const CardImg = styled.div`
-//   width: 272px;
-//   height: 218px;
-//   float: left;
-//   `;
-
 const CardEntry = ({
-  id, name, price, ratingLabel, ratingScore, category, description, neighborhood, photos
-}) => {
-  return (
-    <Card>
-      <PhotosCarousel photos={photos} />
-      <CardText>
-        <Name>{name}</Name>
-        <Details>
-          <span>{category}</span>
-          <span> &#8226; </span>
-          <span>{neighborhood}</span>
-          <span> &#8226; </span>
-          <span>{price}</span>
-        </Details>
-        <div>
-          <img src="https://zigat.s3-us-west-1.amazonaws.com/z-favicon.png" alt="logo" />
-          <span> </span>
-          <RatingLabel>{ratingLabel}</RatingLabel>
-          <span> </span>
-          <RatingScore>{ratingScore}</RatingScore>
-        </div>
-        <Description>{description}</Description>
-      </CardText>
-    </Card>
-  );
+  id, name, price, ratingLabel, ratingScore,
+  category, description, neighborhood, photos, handleClick,
+}) => (
+  <Card>
+    <PhotosCarousel photos={photos} />
+    <CardText onClick={(e) => { handleClick(e, id); }}>
+      <Name>{name}</Name>
+      <Details>
+        {category}
+        <span> &#8226; </span>
+        {neighborhood}
+        <span> &#8226; </span>
+        {price}
+      </Details>
+      <div>
+        <Logo src="https://zagat.com/assets/img/z-logo-icon-red.svg" alt="logo" />
+        <span> </span>
+        <RatingLabel>{ratingLabel}</RatingLabel>
+        <span> </span>
+        <RatingScore>{ratingScore}</RatingScore>
+      </div>
+      <Description>{description}</Description>
+    </CardText>
+  </Card>
+);
+
+CardEntry.propTypes = {
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  price: PropTypes.string.isRequired,
+  ratingLabel: PropTypes.string.isRequired,
+  ratingScore: PropTypes.string.isRequired,
+  category: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  neighborhood: PropTypes.string.isRequired,
+  photos: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Nice! I was running into this issue before and it looks like you have solved it. :)